Gems/ROS2/Code/Source/FactorySimulation/ConveyorBeltComponentKinematic.cpp

```diff
@@ -0,0 +1,306 @@
+/*
```

Comment:
This component should be moved to a separate Gem I believe.

Reply:
Yes, but I belive it should be work for separate PR.